### DISCUSSION TITLE
zellij attach --create session-name with CWD?

### DISCUSSION THREAD

**Discussion by H4ckint0sh** [zellij-org/zellij#2770] [Q&A] [answered]
Is there any way to create the new session with a specific CWD?

**Comment by Lockszmith-GH** [comment, marked as answer]
If you're ok with a solution that isn't 'pure zellij' one - changing the dir before calling zellij should do the trick.

I just tried this in bash and it seems to have worked:

```bash
( cd <NEW CWD LOCATION>; zellij attach --create session-name )
```

Note the `(` `)` - that runs the code inside a sub-shell, so that `cd` doesn't effect the shell you're running from.
